src/configurator/validators/offline.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+const defaultPattern = [ "**/*.*", "!**/*.html", "index.html" ]
+const defaultOfflineConfig = {
+  appcache: true,
+  serviceWorker: true,
+  pattern: defaultPattern,
+}
+
+export default (
+  { config, errors }:
+  { config: StatinamicConfig, errors: Array<string>}
+) => {
+  // Disable offline for development
+  if (config.dev) {
+    config.offline = false
+    return
+  }
+
+  if (!config.offline) {
+    return
+  }
+
+  if (!config.offlineConfig) {
```

Comment:
OK. how about the configurator will convert 

```
offline: Object
```

to 

```
offline: true
offlineConfig: Object
```

This will sastify flow and also a clean logic for the core